test/test_collection.py

```diff
@@ -2253,6 +2253,35 @@ def afind(*args, **kwargs):
         for helper, args in helpers:
             helper(*args, let={})  # type: ignore
 
+    def assertRepr(self, obj):
+        new_obj = eval(repr(obj))
+        self.assertEqual(type(new_obj), type(obj))
+        self.assertEqual(repr(new_obj), repr(obj))
+
+    def test_index_model_repr(self):
```

Comment:
Probably this should at least be organized in a separate test class. Currently it lives in a subclass of `IntegrationTest` which seems unnecessary since it doesn't require database connectivity. As I said before, it seems like a new file `test_operations.py` might be appropriate. Just looking at `operations.py`, I see `SearchIndexModel` which could likely also benefit from `__repr__()` in the future (not sure if Django will use it), but that would be more tests that could live in such a file..